cmd/svcat/svcat_test.go

```diff
@@ -236,7 +236,10 @@ func TestCommandOutput(t *testing.T) {
 		{name: "provision instance", cmd: "provision ups-instance -n test-ns --class user-provided-service --plan default", golden: "output/provision-instance.txt"},
 		{name: "provision instance and wait", cmd: "provision ups-instance -n test-ns --class user-provided-service --plan default --wait", golden: "output/provision-instance-and-wait.txt"},
 		{name: "deprovision instance", cmd: "deprovision ups-instance -n test-ns", golden: "output/deprovision-instance.txt"},
-		{name: "deprovision instance and wait", cmd: "deprovision ups-instance -n test-ns --wait", golden: "output/deprovision-instance-and-wait.txt"},
+
+		// This test does not work when waiting for the instance to not exist and there is no way to fix it in the existing test framework
```

Comment:
Let's remove this comment and text entirely, instead of leaving this in, the new BDD tests that you added is the replacement for these.